vllm/model_executor/layers/linear.py

```diff
@@ -468,13 +468,6 @@ def weight_loader(self,
                     "MergedColumnParallelLinear, assume the weight is "
                     "the same for all partitions.")
 
-        if fp8_scales_shard_indexer is None:
```

Comment:
Can you just explain why this was about to be removed?

Reply:
The models we're getting now have shapes defined for the scales. We were casting the shapes initially since before x.shape returned empty.

Reply:
Makes sense. This is much cleaner